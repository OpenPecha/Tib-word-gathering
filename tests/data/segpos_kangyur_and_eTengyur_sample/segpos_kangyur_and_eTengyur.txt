p1 ༄༅ ། ། <utt>
འདུལ་བ་ ཀ་ བཞུགས་ སོ ། ། <utt>
p2 ln1 ༄༅༅ ། ། <utt>
རྒྱ་གར་ སྐད་ དུ ། <utt>
བི་ན་ཡ་ བསྟུ ། <utt>
བོད་ སྐད་ དུ ། <utt>
