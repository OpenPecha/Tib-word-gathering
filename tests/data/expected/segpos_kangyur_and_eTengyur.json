[
    {
        "source": "༄༅།།",
        "target": "༄༅ ། །"
    },
    {
        "source": "འདུལ་བ་ཀ་བཞུགས་སོ།།",
        "target": "འདུལ་བ་ ཀ་ བཞུགས་ སོ ། །"
    },
    {
        "source": "༄༅༅།།",
        "target": "༄༅༅ ། །"
    },
    {
        "source": "རྒྱ་གར་སྐད་དུ།",
        "target": "རྒྱ་གར་ སྐད་ དུ །"
    },
    {
        "source": "བི་ན་ཡ་བསྟུ།",
        "target": "བི་ན་ཡ་ བསྟུ །"
    },
    {
        "source": "བོད་སྐད་དུ།",
        "target": "བོད་ སྐད་ དུ །"
    }
]
